source/crud/tests/unified/client-bulkWrite-replaceOne-sort.yml

```diff
@@ -0,0 +1,73 @@
+description: client bulkWrite updateOne-sort
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "8.0"
+
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: client bulkWrite replaceOne with sort option
+    operations:
+      - object: client0
+        name: clientBulkWrite
+        arguments:
+          models:
+            - replaceOne:
+                namespace: crud-tests.coll0
```

Comment:
I'll note that this namespace string will still end up hard-coded in the test file since YAML provides no way to concatenate aliases. However, you can define it on first use here and then reference it later in `expectEvents[0].events[0].commandStartedEvent.command.nsInfo[0].ns`.